Description: avoid using npm during build
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2021-01-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@
 install:
 	@rm -rf $(DESTDIR)
 	./tools/install.py $(DESTDIR)
-	npm install --prefix $(DESTDIR)
+#	npm install --prefix $(DESTDIR)
 
 # Create an distributable archive of emscripten suitable for use
 # by end users. This archive excludes node_modules as it can include native
